openlibrary/templates/type/edition/view.html

```diff
@@ -288,10 +288,10 @@
               </div>
             $if is_librarian:
               $if ocaid:
-                <div class="edition-omniline-item">
+                <!-- <div class="edition-omniline-item">
                  <div>$_("Internet Archive")</div>
                   <span><a href="https://archive.org/details/$ocaid">$ocaid</a></span>
-                </div>
+                </div> -->
```

Comment:
Why is this commented out? I'm not sure I understand why this was touched.

Reply:
Oh oops, not quite sure how that happened. I'll revert it back to what it was before